tests/integration/Mvc/ModelCest.php

```diff
@@ -51,6 +54,33 @@ public function executeCamelCaseRelation(IntegrationTester $I)
         $I->assertEquals($expected, $actual);
     }
 
+    /**
+     * @dataProvider findFirstProvider
+     *
+     * @param IntegrationTester $I
+     * @param Example $example
+     */
+    public function findFirstByStringPrimaryKey(IntegrationTester $I, Example $example)
+    {
+        $I->wantToTest('Phalcon\Mvc\Model - findFirst()');
+
+        (new StringPrimaryMigration())($this->container->get('db'));
+        $model = ModelWithStringPrimary::findFirst($example['params']);
+
+        $I->assertSame($example['found'], $model instanceof Model);
```

Comment:
AFAIK, `assertSame` compare by value but `assertEquals` compares identity. Thus,
```php
$I->assertSame(TRUE, 1); //  TRUE
```

IF I'm not wrong, this should use assertEquals, or better:
```php
$I->assertTrue($example['expected'] === $model instanceof Model);
```

Reply:
Nope, it is vice versa.

'2204' !== 2204
assertSame('2204', 2204) // this test fails

'2204' == 2204
assertEquals('2204', 2204) // this test passes

Reply:
Okay, thank you